crates/stackable-operator/src/commons/s3/crd.rs

```diff
@@ -104,55 +98,13 @@ pub enum S3AccessStyle {
 }
 
 /// Set a named AWS region, or defer to an auto-discovery mechanism.
-#[derive(Clone, Debug, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
+#[derive(Clone, Debug, Default, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
```

Comment:
Can you please provide the new `region` CRD snippet? I'm not sure if the defaults are set correctly

Reply:
Your suspicion was right. This indeed does not correctly set the default.

```yaml
region:
  default:
    name: ''
  description: |-
    AWS service API region used by the AWS SDK when using AWS S3 buckets.

    This defaults to `us-east-1` and can be ignored if not using AWS S3 buckets.

    NOTE: This is not the bucket region, and is used by the AWS SDK to construct endpoints for various AWS service APIs. It is only useful when using AWS S3 buckets.
  properties:
    name:
      default: us-east-1
      type: string
  type: object
```

Is there a way to forward this as expected using `schemars`?

Reply:
Nevermind, I was looking at this from the wrong angle. It is fixed in 7627c27.

Reply:
Thanks!